jest.config.js

```diff
@@ -0,0 +1,13 @@
+/* module.exports = {
+	setupFilesAfterEnv: ['./src/setupTests.js'],
+}; */
+
```

Comment:
```suggestion
```
If you don't use it, you can delete it (but to be honest, I am not sure what these lines do if they are not commented out).